lib/driver.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import { BaseDriver } from 'appium-base-driver';
+import log from './logger';
+import WDAMacServer from './wda-mac';
+import { desiredCapConstraints } from './desired-caps';
+import commands from './commands/index';
+
+const NO_PROXY = [
+  ['GET', new RegExp('^/session/[^/]+/appium')],
+  ['POST', new RegExp('^/session/[^/]+/appium')],
+  ['POST', new RegExp('^/session/[^/]+/element/[^/]+/elements?$')],
+  ['POST', new RegExp('^/session/[^/]+/elements?$')],
+];
+
+class Mac2Driver extends BaseDriver {
+  constructor (opts = {}) {
+    super(opts);
+    this.desiredCapConstraints = desiredCapConstraints;
+    this.locatorStrategies = [
+      'id',
+      'name',
+      'accessibility id',
+
```

Comment:
don't see much of a reason for all the spacing?

Reply:
all locators within same groups represent the same WDA locator type. E.g. `id`, `name` and `accessibility id` call the same lookup by identifier on WDA side